pkgs/by-name/ai/airwin2rack/package.nix

```
@@ -53,7 +53,7 @@ let
 
   vcvRackSdk = srcOnly vcv-rack;
   pname = "airwin2rack";
-  version = "2.13.0";
+  version = "0-unstable-2025-04-07";
```

```suggestion
  version = "2.13.0-unstable-2025-04-07";
```
This should be the last release version, since the package had a proper release.